client/src/components/History/CurrentHistory/HistoryDetails.vue

```diff
@@ -11,12 +11,12 @@ import UtcDate from "@/components/UtcDate.vue";
 interface Props {
     history: HistorySummary;
     writeable: boolean;
-    summarized: boolean;
+    summarized?: "both" | "annotation" | "tags" | "none" | "hidden";
```

Comment:
This type is used at least in 2 places, you should consider moving it to a shared type alias to avoid duplication.

Reply:
Moved it to a new `../History/Layout/types` file